### PR TITLE
deprecate riak_kv_backup:backup/3

### PR DESCRIPTION
**Pull request by jrwest**
as discussed.
- [ ] add details to this PR to describe why we are deprecating it
- [ ] backup/restore riak_test is probably going to fall, address this

### CODE REVIEW

File: src/riak_kv_backup.erl [hunk 1]
```diff
@@ -27,15 +27,24 @@
...
 
 %%% BACKUP %%%
 
-%% @doc 
-%% Connect to the cluster of which EntryNode is a member, 
+%% @doc
+%% original entry point for legacy backup, now deprecated.
+%% @see deprecated_backup/3
+backup(_EntryNode, _BaseFilename, _Mode) ->
+    io:format("riak-admin backup has been deprecated and its use is not recommended.~n"
```
Comment (gburd) [inline src/riak_kv_backup.erl:39]: "riak-admin backup has been deprecated in favor of an operational procedure for cluster wide backups detailed in the documentation and codified in various available operational automation tools."

Comment (bookshelfdave) [inline src/riak_kv_backup.erl:39]: riak-admin backup has been deprecated in favor of **AN** operational procedure

Comment (bookshelfdave) [inline src/riak_kv_backup.erl:39]: do we want something like: contact cliserv for more info?

see also: https://github.com/basho/riak_ee/blob/develop/rel/files/riak-repl#L24

**Comment by bookshelfdave** [comment]
fwiw, this seems to work fine:

```
$ ./dev/dev1/bin/riak-admin backup foo bar baz all
riak-admin backup has been deprecated and its use is not recommended.
please see the documentation for more information on how to safely backup a cluster.
```

**Comment by jonmeredith** [comment]
My English teacher would tell me to start a sentence with a capital letter
:) "Please".

On Fri, Feb 21, 2014 at 2:11 PM, Dave Parfitt notifications@github.comwrote:

> fwiw, this seems to work fine:
> 
> $ ./dev/dev1/bin/riak-admin backup foo bar baz all
> riak-admin backup has been deprecated and its use is not recommended.
> please see the documentation for more information on how to safely backup a cluster.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/basho/riak_kv/pull/850#issuecomment-35773969
> .

## 

Jon Meredith
VP, Engineering
Basho Technologies, Inc.
jmeredith@basho.com

**Comment by bookshelfdave** [comment]
I figured the official wording would probably change, hence the lack of a +1.

**Comment by gburd** [comment]
PM +1

**Comment by bookshelfdave** [comment]
I pushed up a small change to capitalize the warning. 

+1

**Pull request merged**